dashboard/views/signup.ejs

```diff
@@ -75,4 +80,10 @@
     </div>
   </div>
 </div>
-<%- include('includes/footer') %>
+<script>
+  $(document).ready(function() {
+    passwordStrength();
+	  confirmPassword();
```

Comment:
Fix indentation.
Use something like `passwordStrength("password", "password-box");` and `confirmPassword("confirm-password", "password");` and use document.getElementById inside the functions.